adaptive_breakpoints/lib/src/adaptive_breakpoints.dart

```diff
@@ -0,0 +1,377 @@
+import 'package:flutter/material.dart';
+
+/// Adaptive Window in Material has five different window sizes. Each one can be
+/// broken into smaller bits.
+///
+/// Extra small represents phones and small tablets on portrait view.
+/// Small represents tablets in portrait view and phones on landscape view.
+/// Medium is going to represent large tablets in landscape view.
+/// Large represents computer screens.
+/// Extra large represents TVs.
+enum AdaptiveWindow {
+  xs,
+  s,
+  m,
+  l,
+  xl,
+}
+
+extension AdaptiveWindowExtension on AdaptiveWindow {
+  String get name {
+    switch (this) {
+      case AdaptiveWindow.xs:
+        return 'xsmall';
+        break;
+      case AdaptiveWindow.s:
+        return 'small';
+        break;
+      case AdaptiveWindow.m:
+        return 'medium';
+        break;
+      case AdaptiveWindow.l:
+        return 'large';
+        break;
+      case AdaptiveWindow.xl:
+        return 'xlarge';
+        break;
+      default:
+        throw AssertionError('Error');
+    }
+  }
+
+  double get longestWidth {
+    switch (this) {
+      case AdaptiveWindow.xs:
+        return 599;
```

Comment:
super nit: you can omit the `break`s and even have the return on the same line as the `case`

Reply:
Removed breaks but the return statements are still on their own line due to the flutter formatter.